src/app/forms/forms.demo.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-forms-demo",
+    styles: [`.list-nav { columns: 3; margin-bottom: 20px; }`],
+    template: `
+        <h2>Forms</h2>
+        <ul class="list-nav">
+            <li><a [routerLink]="['./input-vertical']">Input Vertical</a></li>
+            <li><a [routerLink]="['./input-horizontal']">Input Horizontal</a></li>
+            <li><a [routerLink]="['./input-compact']">Input Compact</a></li>
+            <li><a [routerLink]="['./input-vertical']">Input Vertical Grid</a></li>
+            <li><a [routerLink]="['./input-horizontal']">Input Horizontal Grid</a></li>
+            <li><a [routerLink]="['./input-compact']">Input Compact Grid</a></li>
```

Comment:
same for these two.

Reply:
Would be able to update: https://clarity-forms.surge.sh/ with the correct demos so that I can review the correct demos?

Reply:
Updated